muesli/web/viewsExam.py

```diff
@@ -554,188 +573,44 @@ def enterPointsSingle(request):
     exam = request.context.exam
     exercises = exam.exercises
     request.javascript.append('prototype.js')
-    request.javascript.append('scriptaculous/scriptaculous.js')
```

Comment:
I'm not sure what this library does. Can you explain that to me? If you really remove it, please also remove the Symlinks and all notes that this is a requirement.

Reply:
This library was used for the autocompletion, wich is now replaced with the select2-dropboxes. The library will be removed in an upcoming commit.